packages/react/src/components/Authenticator/ResetPassword/ConfirmResetPassword.tsx

```diff
@@ -53,9 +62,6 @@ export const ConfirmResetPassword = (): JSX.Element => {
             onClick={() => {
               send({
                 type: 'RESEND',
-                data: {
-                  username: state.context.username,
-                },
```

Comment:
Why was this removed? There is no `username` field on this page, so the `username` won't exist in the `formValues` context. We need to be able to pass this up to properly submit the `RESEND` request.

Reply:
We don't need to send `username` anywhere because it's persisted in `context.username` anyway. Instead of preparing data here, we're directly using context in this service call: https://github.com/aws-amplify/amplify-ui/blob/f63cfeb4f6bf6fe8fa95929c5397d84703e922ef/packages/core/src/machines/actors/auth/resetPassword.ts#L107-L108